_release/bat/release_test.py

```diff
@@ -405,10 +405,6 @@ def test_start_all_workloads(self):
         """Start one instance of all workloads"""
         self.failUnless(launch_all_workloads())
 
-    def test_start_all_workloads10(self):
```

Comment:
How about we call this "test_start_all_cattle10()" and have that only launch the cattle type workloads and not ones that are going to amount to download image from network'd image store, copy/convert it to a qemu-img, upload to network'd volume store, and boot from volume.  Meaningful cattle aren't going to do a synchronous conversion in their boot flow.